conjure-java-core/src/main/java/com/palantir/conjure/java/util/PrimitiveHelpers.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.util;
+
+import com.squareup.javapoet.AnnotationSpec;
+import com.squareup.javapoet.TypeName;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public final class PrimitiveHelpers {
+
+    private PrimitiveHelpers() {}
```

Comment:
Pedantic naming nit: Rather than `Helper` or `Util` prefixes, we generally use plural of the target, e.g. `Primitives`.